### ISSUE TITLE
Sent loki from wallet to excange but it's pending for 24h!

### ISSUE THREAD

**Issue by cekikv** [loki-project/loki#187]
I sent my LOKI from wallet to excange. 24h passed and still nothing. What went wrong?

![loki](https://user-images.githubusercontent.com/35508846/44574476-cc046b80-a789-11e8-8acf-77defd955d42.JPG)

**Comment by KeeJef** [comment]
You are using the wrong version of the Loki software, the official version is 0.2.0 you are using 0.1.0 and are stuck on the wrong version of the blockchain. please update to 0.2.0 by downloading the new version from our website

**Comment by cekikv** [comment]
Thanks for quick response! I downloaded and now can't connect wallet. It says Deamon Failed  to start. Please check your wallet and deamon log for errors. You can also start lokid.exe manually.

**Comment by KeeJef** [comment]
Please check that your anti virus is not flagging or deleting lokid.exe sometimes there is a false detection which can cause these issues

**Comment by cekikv** [comment]
Yep I also thought before clicked comment ... Windows defender .. It's in sync phase now .. I'll let you know if everything is ok.
Thanks,

**Comment by cekikv** [comment]
Will this be any kind of problem coz i sent LOKI from older version of wallet ?

**Comment by cekikv** [comment]
It's pretty much the same except it's 0.2.0 version of wallet .. Do I need to wait more ?

![loki](https://user-images.githubusercontent.com/35508846/44591570-5e266700-a7be-11e8-88c1-b495484030f9.PNG)

**Comment by KeeJef** [comment]
No, your transfer should be working fine now that you are on the correct chain

**Comment by cekikv** [comment]
Update! it's working now. Thank you for the support.
